Update package definition

kevinrizza opened the pull request:
Update fork so that it can be directly imported
Replace package definition from github.com/irifrance/gini
to github.com/operator-framework/gini

Signed-off-by: kevinrizza <krizza@redhat.com>

timflannagan commented on go.mod:
```diff
@@ -1 +1 @@
-module github.com/irifrance/gini
+module github.com/operator-framework/gini
```
This looks off - you may need to re-run the tide/vendor commands.

timflannagan commented:
I was able to find a good number of package imports that still need to be remapped which would fail:

```bash
$ grep -rnw . -e "github.com/irifrance/gini/z" | wc -l
88
```

kevinrizza committed:
Update package definition

Update fork so that it can be directly imported
Replace package definition from github.com/irifrance/gini
to github.com/operator-framework/gini

Signed-off-by: kevinrizza <krizza@redhat.com>
